Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -275,6 +275,12 @@ const ScrollView = createReactClass({
      * @platform ios
      */
     minimumZoomScale: PropTypes.number,
+     /**
+     * Enables nested scrolling for Android API level 21+.
+     * Nested scrolling is supported by default in iOS platforms
```

Comment:
`Nested scrolling is supported by default on iOS` sounds better I think.

Reply:
👍  983e69f